### PR TITLE
Feature/search bar

### PR DESCRIPTION
**Pull request by MarkRivera**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by jlong5795 — APPROVED** [review]

File: src/components/graphs/GraphContainer.js [hunk 1]
```diff
@@ -18,14 +18,26 @@ import Pagination from './pagination/Pagination';
...
+) => {
   // Only grab events that have occurred in the last 12 months and filter out any that do not have dates:
   const start = today - elevenMonths;
+  let dateFilter;
 
-  const dateFilter = data.map(incident => {
-    incident.date = Date.parse(incident.date);
-    return incident;
-  });
+  !dateIsMilli
```
Comment (jstewart8053) [inline src/components/graphs/GraphContainer.js:33]: Can you use this before declaring it? I see it in the useState hook below, but I thought the hook would have to come first.

Comment (jstewart8053) [reply src/components/graphs/GraphContainer.js:33]: I guess I'm wondering if all your state should be at the top of the page?

Comment (jlong5795) [reply src/components/graphs/GraphContainer.js:33]: What is being used before declaration?

Comment (jstewart8053) [reply src/components/graphs/GraphContainer.js:33]: !dateIsMilli unless I'm reading it incorrectly

Comment (jstewart8053) [reply src/components/graphs/GraphContainer.js:33]: Nevermind. It is part of the filter const.

Comment (jstewart8053) [reply src/components/graphs/GraphContainer.js:33]: But should the state live at the top of the page?

Comment (jlong5795) [reply src/components/graphs/GraphContainer.js:33]: That is part of a helper function that lives outside of the component. At run time it exists inside the correct scope

Comment (jstewart8053) [reply src/components/graphs/GraphContainer.js:33]: Gotcha Jason. Thanks for explaining.

**Review by jstewart8053 — APPROVED** [review]

**Pull request merged**